3p/twitter.js

```diff
@@ -97,6 +109,31 @@ export function twitter(global, data) {
   }
 }
 
+/**
+ * @param {*} momentid
+ * @visibleForTesting
+ */
+export function cleanupMomentId_(momentid) {
```

Comment:
both tweet and moment ids are always int64 ( https://developer.twitter.com/en/docs/basics/twitter-ids ). Unfortunately initially we did not restrict data-tweetid to that and people started putting junk in there so we had to add this cleaup method to be more graceful with mistakes. (we could not change the validation rule for backward compatibility)

Since momentid is new, let's just restrict it to numbers via validation rule and remove this all together.

Reply:
Sounds good, made the change.